src/main/webapp/i18n/de/exam.json

```diff
@@ -59,6 +59,8 @@
             }
         },
         "examScores": {
+            "xAxes": "Ergebnis in %",
+            "yAxes": "Anzahl an TeilnehmerInnen",
```

Comment:
We use just plural `Teilnehmer` (= `Anzahl an Teilnehmern`) in other places:
https://github.com/ls1intum/Artemis/blob/71c154e3360777db1be0a118b27169004528ff6b/src/main/webapp/i18n/de/exam.json#L69
Not sure what the guideline here is.

Reply:
> When switching between languages the label texts do not update, only when reloading the page.

Not worth solving this imo. Who switches their language constantly? If you reload the page it works

Reply:
Ok. Lgtm then.

Reply:
> We use just plural `Teilnehmer` (= `Anzahl an Teilnehmern`) in other places:
> https://github.com/ls1intum/Artemis/blob/71c154e3360777db1be0a118b27169004528ff6b/src/main/webapp/i18n/de/exam.json#L69
> 
> 
> Not sure what the guideline here is.

Changed it